script/Ciel/Day 13/13_01_02_CIEL13_6_1.txt

```diff
@@ -375,7 +376,7 @@ it hurts so much that i cant stand it so   this is where i end
 [sha:e9acdd2430ca50c7a9c7715ba132e937d2dfe785]{
 -- Page 180, Offset 35786.
 -- そのまえに、もうひとり、わるいヤツをたいじします。
-but before i go theres one more evildoer that i need to eliminate
+but before i go theres one more evildoer that i need to deal with
```

Comment:
go, there's

Reply:
This and some of the later ones are all hiragana sentences, which we've been styling as all lowercase  o punctuation

Reply:
I see that but:
1. the source text has punctuation
2. removing commas changes how a sentence reads. If we want to get rid of apostrophes and periods that's fine but commas I think should stay

Reply:
I guess what I've done elsewhere is three spaces in lieu of a full stop, that work for you?

Reply:
I guess I'm ok with that